src/bsmHandler.cpp

```diff
@@ -376,40 +376,25 @@ bool BSMHandler::process( const std::string& bsm_json ) {
     return result_ == ResultStatus::SUCCESS;
 }
 
-void BSMHandler::handlePartIIRedaction(rapidjson::Value& data) {
-    bool debug = false;
-    
+void BSMHandler::handlePartIIRedaction(rapidjson::Value& data) {    
     int numMembersRedacted = 0;
 
     // check if partII redaction is required
     if (data.HasMember("partII") && is_active<kPartIIRedactFlag>()) {
-        if (debug) { std::cout << "partII redaction is required" << std::endl; }
```

Comment:
Should we call the logging debug instead of removing these logging statements?

Reply:
The logger isn't accessible from this class at this time. I can look into passing in the pointer when the BSMHandler is created, though!

Reply:
A PpmLogger pointer is now passed to the BSMHandler class whenever it is instantiated and this is used when partII redaction is required.